src/main/java/com/google/codeu/data/Country.java

```diff
@@ -1,16 +1,23 @@
 package com.google.codeu.data;
 
+import java.util.HashSet;
+import java.util.Set;
+
 public class Country {
   private String code;
   private String name;
   private double lat;
   private double lng;
+  Set<String> categories = new HashSet<String>();
 
   public Country(String code, String name, double lat, double lng) {
     this.code = code;
     this.name = name;
     this.lat = lat;
     this.lng = lng;
+    categories.add("Food");
+    categories.add("Attractions");
+    categories.add("Culture");
```

Comment:
This makes sense for now, but if we would change the categories to be added by users (and not initialize these three) do you have any idea where we would create them? Asking for my own curiosity, this seems fine for the scope of this PR

Reply:
I'm thinking maybe in another PR that implements the front end for posting messages to a specific country, it will list the categories as buttons (kinda like tags) by iterating through the categories set. There can be an extra button to create a category that isn't already listed. Maybe it can use a modifier method we can add that appends a category to that country's categories set.

Reply:
Sounds good :D